unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Passport/Passport/Popups/OpenUrlView.cs

```diff
@@ -0,0 +1,51 @@
+using DCL.Helpers;
+using DCL.Interface;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class OpenUrlView : MonoBehaviour
```

Comment:
Why are we not implementing this view as an UI Component? I'm seeing you're implementing, for example, the management of a `ShowHideAnimator` to show/hide the view, and it is something that we have already implemented in the `BaseComponentView` class.

Reply:
I have no idea on why I didn't use it, changing it now :D